### DISCUSSION TITLE
Testing NIDS Rules not producing alerts

### DISCUSSION THREAD

**Discussion by hotcobra** [Security-Onion-Solutions/securityonion#12562] [2.4] [answered]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

6

### RAM

16gig

### Storage for /

200g

### Storage for /nsm

default partition

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I'm reasonably new to Security Onion but  trying to Test NIDS rules per documentation (https://docs.securityonion.net/en/2.4/managing-alerts.html).  I used procedures listed in NIDS Testing on my Security Onion VM, the host it's running on and other MACOS hosts.  I don't get any alerts from it.  Otherwise, I am getting at least 1 valid alert from my network (Rule 2027397).

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment, marked as answer]
Are you ingesting the traffic from your test machine that is running the ```curl testmynids.org/uid/index.html``` command? 

These are the two alerts I got from running the above command in powershell. That machines traffic goes through a network tap and is ingested by my Security Onion box.
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/94730068/59ebb722-7e36-4eb7-99af-69b1ac1e5fc0)

**Reply by hotcobra** [reply]
You're right.  After a using NC and tcpdump, my switch is not mirroring all traffic to the SO box.
